concept_formation/cobweb3.py

```diff
@@ -18,6 +18,10 @@
 from concept_formation.cobweb import CobwebTree
 from concept_formation.continuous_value import ContinuousValue
 from concept_formation.utils import isNumber
+from concept_formation.utils import weighted_choice
+from concept_formation.utils import most_likely_choice
+
+continuous_value = "#ContinuousValue#"
```

Comment:
Might want to name this continuous_value_tag or something to cut down on confusion with other places we use ContinuousValue particularly for importing.